etc/vale.ini

```diff
@@ -0,0 +1,30 @@
+# Example Vale config file (`.vale.ini` or `_vale.ini`)
```

Comment:
hello, is it possible to add Copyright there as well ?

Reply:
Yes will do.